critiquebrainz/frontend/forms/utils.py

```diff
@@ -0,0 +1,9 @@
+import pycountry
+from babel.core import UnknownLocaleError, Locale
+
+
+def get_language_name(language_code):
```

Comment:
This should have some documentation that explains what it does, and why there are the two different versions. Include some examples of how it works in each case, and why the first version might fail.

We should also have some tests, which specifically check our `no` bug.

Does it make sense to put this in `forms/utils.py`? I'm just thinking because we load it in `frontend/__init__.py`. Maybe there's a more generic frontend utils package where we could put it.

Reply:
Yeah, actually regarding this I think `babel` or `pycountry` should be removed from this part. I do not think we should be doing using both.

Reply:
The issue arises from https://github.com/metabrainz/critiquebrainz/blob/2d7a0c1958b5bd221b4340efee30123b4bcb66a7/critiquebrainz/db/review.py#L29-L32. Here, the language codes are stored in a list from pycountry. While getting the language name, we try first babel then pycountry. I think we should drop the `babel` part from this method.